src/plugins/shared-variables/graph/plotted-variables-adornment/plotted-variables-adornment-component.tsx

```diff
@@ -58,21 +58,76 @@ export const PlottedVariablesAdornmentComponent = observer(function PlottedVaria
     const tPixelMax = xScale(xMax);
     const kPixelGap = 1;
     for (const instanceKey of model.plottedVariables.keys()) {
+      const plottedVar = model.plottedVariables.get(instanceKey);
+      const values = plottedVar?.variableValues;
       const tPoints = model.computePoints({
         instanceKey, min: tPixelMin, max: tPixelMax, xCellCount, yCellCount, gap: kPixelGap, xScale, yScale
       });
       if (tPoints.length > 0) {
         const path = `M${tPoints[0].x},${tPoints[0].y},${curveBasis(tPoints)}`;
 
         const selection = select(plottedFunctionRef.current);
-        selection.append("path")
-          .attr("class", `plotted-function plotted-function-${classFromKey}`)
-          .attr("data-testid", `plotted-function-path${classFromKey ? `-${classFromKey}` : ""}`)
-          .attr("stroke", graphModel.getColorForId(instanceKey))
-          .attr("d", path);
+        const traceGroup = selection.append("g")
+          .attr("class", 'plotted-variable')
+          .on('mouseover', function(d, i) { this.classList.add('selected'); })
+          .on('mouseout', function(d, i) { this.classList.remove('selected'); });
+
+        // Highlight of line (visible on mouseover)
+        traceGroup.append('path')
+          .attr('class', 'plotted-variable-highlight plotted-variable-highlight-path')
+          .attr('d', path);
+        // Path for main line
+        traceGroup.append('path')
+          .attr('class', `plotted-variable-path`)
+          .attr('stroke', graphModel.getColorForId(instanceKey))
+          .attr('d', path);
+        if (values) {
+          const x = model.pointPosition(values.x, xScale, xCellCount),
+            y = model.pointPosition(values.y, yScale, yCellCount),
+            textHeight = 12,
+            padding = 4,
+            offsetFromPoint = 14,
+            labelFormat = format('.3~r'),
+            label = `${labelFormat(values.x)}, ${labelFormat(values.y)}`;
+          // Highlight for value marker
+          traceGroup.append('circle')
+            .attr('class', 'plotted-variable-highlight plotted-variable-highlight-value')
+            .attr('r', graphModel.getPointRadius() + 2.5)
+            .attr('stroke-width', '5')
```

Comment:
I think you should use a constant for the `highlightStrokeWidth` and then use that to define `r` as well.